### ISSUE TITLE
Adding input constraints to question statement?

### ISSUE THREAD

**Issue by Selbl** [Open-Deep-ML/DML-OpenProblem#117]
I noticed that some of the questions could greatly benefit from clearly identifying input constraints in the input statement. For example Q4 - Calculate Mean by Row or Column has an input "mode" which is a string, which should ideally be equal to either 'column' or 'row'. The official solution also considers the case where the mode is neither column nor row, and raises a value error. However, none of the test cases have an input of 'mode' that is not 'row' or 'column'.

Knowing beforehand which values of 'mode' the code will be tested against, would be helpful in determining how to code the question. It would also be useful to indicate what the code's behaviour should be with unexpected inputs to give guidance to coders on how to deal with those cases.

**Comment by moe18** [comment]
I don't think that is needed for this problem, but thank you for the suggestion, I just think that it would add a layer of complexity we dont need to test for
